bot/exts/github/_source.py

```diff
@@ -0,0 +1,44 @@
+from inspect import getsourcelines
+from typing import Optional, Union
+
+import discord
+from aiohttp import ClientSession
+from discord import Embed
+from discord.ext.commands import Command
+
+
+class Source:
+    """Displays information about the bot's source code."""
+
+    def __init__(
+        self, http_session: ClientSession, bot_avatar: discord.asset.Asset
+    ) -> None:
+        self.http_session = http_session
+        self.max_field_length = 500
```

Comment:
Let's name it in uppercase since it's a constant.
```suggestion
        self.MAX_FIELD_LENGTH = 500
```